specification/imagebuilder/resource-manager/readme.go.md

```diff
@@ -14,6 +14,7 @@ These settings apply only when `--go` is specified on the command line.
 batch:
   - tag: package-2018-02
   - tag: package-2019-02
+  - tag: package-2019-05
```

Comment:
According to the README.md file the tag is `package-preview-2019-05`

Reply:
thanks. fixed the tag for go sdk